react-components/src/architecture/concrete/pointsOfInterest/PointsOfInterestDomainObject.ts

```diff
@@ -164,4 +171,30 @@ export class PointsOfInterestDomainObject<PoiIdType> extends VisualDomainObject
       (pois) => pois.status === PointsOfInterestStatus.PendingDeletion
     );
   }
+
+  public getScene(): DmsUniqueIdentifier | undefined {
+    return this._currentScene;
+  }
+
+  public async setCurrentScene(scene: DmsUniqueIdentifier | undefined): Promise<void> {
+    if (scene === this._currentScene) {
+      return;
+    }
+
+    this._currentScene = scene;
+    this._pointsOfInterest = [];
+
+    if (scene === undefined) {
+      return;
+    }
```

Comment:
Is it possible to move this early return along with the first one on the top of we need to have the internal `this._currentScene` and the `this._pointsOfInterest `reset even when the `scene ` is `undefined` and not when the `scene `is equal to `this._currentScene`?

Reply:
We need to reset `this._currentScene` and the points when the scene is undefined. So I'd like to keep it here